meta/main.yml

```diff
@@ -18,3 +19,9 @@ galaxy_info:
     - 'icinga2'
     - 'monitoring'
     - 'icingaweb2'
+  github_branch: master
+
+collections:
+  - ansible.posix
+  - community.general
+  - robertdebock.roles
```

Comment:
Please be aware that `ansible-galaxy` does not support declaring dependencies between roles and collections. These collections therefore need to be added manually to a project's `requirements.yml` (see ansible/ansible#76030). I think we should add a note to the README that dependencies on collections are required.

Reply:
Done in https://github.com/adfinis/ansible-role-icinga2_web/pull/35/commits/f774d88c5102914c1c7b5e2ec48d522d9c18e71f

Can you check if that suits you well?